modules/data_release/ajax/AddPermission.php

```diff
@@ -12,25 +12,84 @@
  *  @link     https://github.com/aces/Loris
  */
 
-$DB =& Database::singleton();
+$user     = User::singleton();
```

Comment:
I might be wrong / this might not make sense for this script but looking at recent changes to the code, we've been doing a lot of `$user = \User::singleton()` instead

Reply:
Should make no difference with this script but I'll do it to be consistent!